### ISSUE TITLE
[monitor-docker]  Error reading monitor statistics: TypeError: Cannot read properties of undefined (reading 'usage')

### ISSUE THREAD

**Issue by aronaks** [hyperledger/caliper#1309]
### Which Caliper version are you using?

v0.5.0-unstable-20220408135025

### Which Node.JS version are you using?

v17.8.0

### Which operating system are you using?

macOs Monterey 12.2.1

### Please provide some context for your error. For example, when did the error occur? What were you trying to achieve, and how?

I tried to monitor remote docker containers. 

npx caliper launch manager --caliper-bind-sut fabric:latest-v2 --caliper-workspace ./ --caliper-bind-cwd ./ --caliper-networkconfig networks/networkConfig.yaml --caliper-benchconfig benchmarks/config.yaml --caliper-flow-only-testcat

### What was the observed incorrect behavior?

Caliper cannot gather monitoring information because of the error

### Please provide the error logs and their surroundings.

```text
[monitor-docker]  Error reading monitor statistics: TypeError: Cannot read properties of undefined (reading 'usage')
```

### Please provide your benchmark configuration file content, if possible.

```yaml
test:
  name: initial try
  description: it is just a proof of concept 
  workers:
    type: local
    number: 1
  rounds:
    - label: main
      description: fixed
      txDuration: 60
      rateControl:
        type: fixed-rate
        opts:
          tps: 5
      workload:
        module: workload/createIdentity.js
monitors:
  resource:
  - module: docker
    options:
      interval: 5
      containers:
        - dev-peer0.org2.example.com-identity_1.0-42252d58153bcf4ee71291af75bf27a4575826b9fe0d5596e34735f00db464d8
        - dev-peer0.org1.example.com-identity_1.0-42252d58153bcf4ee71291af75bf27a4575826b9fe0d5596e34735f00db464d8
```

### Please provide your network configuration file content, if possible.

```yaml
-
```

### Please provide your workload module content, if possible.

```nodejs
-
```

**Comment by fraVlaca** [comment]
Node 17 is not supported. Please let us know when using node 16 the problem persists.

**Comment by aronaks** [comment]
Thanks for the response. I tried with node 16 and node 14. The error is basically the same:

`2022.04.13-11:55:00.263 error [caliper] [monitor-docker] 	Error reading monitor statistics: TypeError: Cannot read property 'usage' of undefined`

**Comment by fraVlaca** [comment]
Thanks! I have just tryied now as well and got the same error ...investigating now, will let you know asap.

**Comment by fraVlaca** [comment]
I recreated the scenario and it dos looks it is a problem on how system informations capturates docker metrics for the docker monitor.
I tried updating the dependency but i got the same error.
It seems that some time it is not able to retrieve certain stats. Which could suggest a problem on docker actually having a problem exporting them.
I will do further invastigations.
For now i'll be updating the way the docker monitor capture docker metrics by using dockerode(which was already used when working with remote nodes) to be use also locally. 
If then the problem persists it will probably mean that the cause of the error would be related on a bug on docker itself rather than caliper
